nullaway/src/main/java/com/uber/nullaway/fixserialization/location/SymbolLocation.java

```diff
@@ -37,32 +37,23 @@ public interface FixLocation {
   String tabSeparatedToString();
 
   /**
-   * Creates header of an output file containing all {@link FixLocation} written in string which
+   * Creates header of an output file containing all {@link SymbolLocation} written in string which
    * values are separated tabs.
    *
    * @return string representation of the header separated by tabs.
    */
   static String header() {
-    return "location"
-        + '\t'
-        + "class"
-        + '\t'
-        + "method"
-        + '\t'
-        + "param"
-        + '\t'
-        + "index"
-        + '\t'
+    return "kind" + '\t' + "class" + '\t' + "method" + '\t' + "param" + '\t' + "index" + '\t'
```

Comment:
Was this auto-formatted? I think it looks better in the previous/existing code (unless GJF recommends this style)

Reply:
Since Java 1.8 there's `String#join(CharSequence delimiter, CharSequence... elements)`, so another option would be:
```
String.join("\t", "kind", "class", "method", "param", "index", "uri")
```

Reply:
Right. @nimakarimipour, probably a good idea to do this in all the places we are performing these tab-separated string concatenations.

Reply:
a8fdd11f30cc3bf3d74189b437a7cac6555d0ddb

@Stephan202 thank you for the suggestion :)
@lazaroclapp replaced all multiple calls with single `join` call.